integration/kubernetes/cleanup_env.sh

```diff
@@ -35,3 +35,6 @@ sudo ip link del flannel.1
 
 # Check no kata processes are left behind after reseting kubernetes
 check_processes
+
+# Checks that pods were not left
+check_pods
```

Comment:
nice - I presume this will not check on all k8s tests we run?

Reply:
@grahamwhaley yes it will

Reply:
Oops, I meant 'will check' - good !